src/components/KTable/ColumnVisibilityMenu.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="k-table-column-visibility-menu">
+    <KDropdown>
+      <KTooltip text="Show/Hide Columns">
```

Comment:
suggestion: we probably want to make tooltip text a prop (possible localization considerations?). Also I like sentence case better (_Show/hide columns_) but that might be just a personal preference.

Reply:
This component won't be exposed. We would end up needing the prop added to `KTable` which I don't think we want. Thoughts @adamdehaven ?

Reply:
I believe we have other hard-coded text already in Kongponents? If yes I'd say just leave it hard-coded so we don't have prop-drilling 2-3 levels deep

Reply:
Yes we do. Okay, since this component won't be exported it's okay to leave it as is for now (or would be an overkill to add a prop just for this).